src/services/defaultClasses.js

```diff
@@ -18,6 +18,10 @@ export default {
         'vue-ads-bg-teal-100': true,
         'vue-ads-bg-gray-100': false,
     },
+    exactMatch: {
+        'bg-yellow-500': true,
+        'cell-exact-match': true,
```

Comment:
These need `vue-ads-` prefix. I'm not sure any styles without it will be exported.